docs/docs/contextual-conversations.mdx

```diff
@@ -1,120 +1,182 @@
 
-```md
-## Story from conversation with a2baab6c83054bfaa8d598459c659d2a on November 28th 2019
-* greet
-  - action_greet_user
-  - slot{"shown_privacy":true}
-* ask_whoisit
-  - action_chitchat
-* ask_whatspossible
-  - action_chitchat
-* telljoke
-  - action_chitchat
-* how_to_get_started{"product":"x"}
-  - slot{"product":"x"}
-  - utter_explain_x
-  - utter_also_explain_nlucore
-* affirm
-  - utter_explain_nlu
-  - utter_explain_core
-  - utter_direct_to_step2
+```rasa-yaml
+stories:
+- story: Story from conversation with a2baab6c83054bfaa8d598459c659d2a on November 28th 2019
+  steps:
+  - checkpoint: STORY_START
+  - intent: greet
+  - action: action_greet_user
+  - slot_was_set:
+    - shown_privacy: true
+  - intent: ask_whoisit
+  - action: action_chitchat
+  - intent: ask_whatspossible
+  - action: action_chitchat
+  - intent: telljoke
+  - action: action_chitchat
+  - intent: how_to_get_started
+    entities:
+    - product: x
+  - slot_was_set:
+    - product: x
+  - action: utter_explain_x
+  - action: utter_also_explain_nlucore
+  - intent: affirm
+  - action: utter_explain_nlu
+  - action: utter_explain_core
+  - action: utter_direct_to_step2
 ```
 
 Here we can see the user has completed a few chitchat tasks first, and then ultimately
-asks how they can get started with Rasa X. The TEDPolicy correctly predicts that
+asks how they can get started with Rasa X. The `TEDPolicy` correctly predicts that
 Rasa X should be explained to the user, and then also takes them down the getting started
 path, without asking all the qualifying questions first.
 
-Since the ML policy generalized well in this situation, it makes sense to add this story
-to your training data to continuously improve your bot and help the ML generalize even
+Because the machine-learning policy has generalized to this situation, you should add this story
+to your training data to continuously improve your bot and help the model generalize
 better in future. [Rasa X](https://rasa.com/docs/rasa-x/) is a tool that can help
 you improve your bot and make it more contextual.
+
+
+#### Setting `max_history`
+
+Usually, only a certain amount of context is relevant to your assistant.
+[`max_history`](policies.mdx#max-history) is a hyperparameter for Rasa dialogue management policies
+that controls how much dialogue history the model looks at to decide which
+action to take next.
+
+You can set the `max_history` by passing it to your policy's `Featurizer`
+in the policy configuration yaml file, for example:
+
+```
+policies:
+  - name: "TEDPolicy"
+    featurizer:
+    - name: MaxHistoryTrackerFeaturizer
+      max_history: 5
+```
+
+You want to make sure `max_history` is set high enough
+to account for the most context your assistant will need to make an accurate
+prediction about what to do next.
+For more details see the docs on [conversation featurization](policies.mdx#conversationfeaturization).
```

Comment:
i think if we're going to cover max_history then we should actually explain what it pays attention to.

Reply:
right now we're just throwing a random number at them without much info